bench/app.py

```diff
@@ -310,16 +318,28 @@ def get_app(
 	If the bench_path is not a bench directory, a new bench is created named using the
 	git_url parameter.
 	"""
-	from bench.bench import Bench
 	import bench as _bench
 	import bench.cli as bench_cli
+	from bench.bench import Bench
+	from bench.utils.app import check_existing_dir
 
 	bench = Bench(bench_path)
 	app = App(git_url, branch=branch, bench=bench)
 	git_url = app.url
 	repo_name = app.repo
 	branch = app.tag
 	bench_setup = False
+	frappe_path, frappe_branch = None, None
+
+	if resolve_deps:
```

Comment:
Make behaviour consistent for command with and without `--resolve-deps`

Reply:
This is just to provide support for frappe versions (when that's mentioned in the hooks of the required apps).